keel-sql/src/main/resources/db/changelog/20200109-fix-delivery-artifact-uniqueness.yml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: fix-delivery-artifact-uniqueness
+      author: emjburns
+      changes:
+        - addColumn:
+            tableName: delivery_artifact
+            columns:
+              - column:
+                  name: fingerprint
+                  type: varchar(255)
```

Comment:
Since sha1 digests are of fixed length, it would be more efficient to make this a `char` of that exact length.